cl/scrapers/management/commands/cl_scrape_opinions.py

```diff
@@ -165,14 +185,21 @@ def save_everything(
         citation.cluster_id = cluster.pk
         citation.save()
 
+    if opinion.author_str:
+        candidate = async_to_sync(lookup_judges_by_messy_str)(
+            opinion.author_str, docket.court.pk, cluster.date_filed
+        )
+        if candidate:
+            opinion.author = candidate[0]
```

Comment:
Dont save if we find more than one judge please

Reply:
@flooie I updated the PR and it is passing tests